AutomaticWorkAssignment/Source/PawnFitness/FormulaPawnFitness.Parser.Tokenize.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace Lomzie.AutomaticWorkAssignment.PawnFitness
+{
+    public partial class FormulaPawnFitness
+    {
+        internal partial class Parser
+        {
+            internal interface IToken { }
+            internal enum Operator
+            {
+                OpenGroup,
+                CloseGroup,
+                ArgSep,
+                Sum,
+                Subtract,
+                Factor,
+                Divide,
+                Modulus,
+                Exp,
+                Root,
```

Comment:
Probably overkill but it was a good way to test some situations in the tokenizer (multi-char operator) or the parser.